text/9999-event-vision.md

```diff
@@ -0,0 +1,60 @@
+# Event basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to write down and agree on a few basics.
+
+### What are events?
+
+Events are a type of log which has a required event name and a specific structure which is implied by that event name.
+
+The event name and its implied structure make events much more suitable for observability compared to traditional logs.
+
+Because of this, the OpenTelemetry project wants to encourage the transition from traditional logs to events.
+
+### OTLP
+
+Since events are a type of log, they share the same OTLP data structure and OTLP pipeline.
+
+### API
+
+OpenTelemetry should have an Event API. This will help to promote the distinction between events and traditional logs,
+and encourage the use of events over traditional logs.
```

Comment:
👍 

While I understand that its possible to use existing logging APIs to emit records shaped like OpenTelemetry events, I don't think it's good enough. User's would have to call logging APIs in a very specific way, and veering off course yields traditional logs which we want to discourage.

With a dedicated OpenTelemetry Event API, we ensure we control our own destiny. We can design an API exactly the way we see events, with no [impedance mismatch](https://www.geeksforgeeks.org/impedance-mismatch-in-dbms/). We have a very clear API story - with first class user facing APIs for traces, metrics, and events (logs), all suitable for using directly in native instrumentation.

Reply:
I think I captured your points here in https://github.com/open-telemetry/oteps/pull/265/commits/e9fce2245c1482225684a7cb33579460b6ecf5e0